extensions/amp-ad/0.1/amp-ad-cross-domain-iframe-handler.js

```diff
@@ -107,6 +106,9 @@ export class AmpAdApiHandler {
       // If support render-start, create a race between render-start no-content
       this.adResponsePromise_ = listenForOncePromise(this.iframe_,
```

Comment:
I added a `renderStart` function, and with the `noContent` function it should be more clear now.